packages/mdc-chips/chip-set/component.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+import {MDCComponent} from '@material/base/component';
+import {MDCChip, MDCChipFactory, MDCChipFoundation} from '../chip';
```

Comment:
I instinctively wanted to suggest `chip/index` here... but I guess this is safe because the subdirectory shouldn't be subject to package.json `main` resolution?

Reply:
Correct, it should be safe. But we can keep `/index` for consistency if you prefer. WDYT?

Reply:
I'm slightly inclined towards consistency so it doesn't trip up any one else reading it, but I dunno if that's just me.

Reply:
I think if we're questioning it, we should lean towards being more explicit. Its not going to hurt to add the `/index`.

Reply:
Done